src/core/control/pagetype/PageTypeMenu.cpp

```diff
@@ -127,7 +127,14 @@ void PageTypeMenu::addMenuEntry(PageTypeInfo* t) {
                          }
 
                          for (MenuCallbackInfo& info: self->menuInfos) {
+
                              if (info.entry == togglebutton) {
+                                 // Disable "Apply to current page" if current format is "Copy."
+                                 if (info.info->page.format == PageTypeFormat::Copy) {
+                                     gtk_widget_set_sensitive(self->menuEntryApply, FALSE);
+                                 } else {
+                                     gtk_widget_set_sensitive(self->menuEntryApply, TRUE);
+                                 }
```

Comment:
nit: this can be simplified to
```suggestion
                                 gtk_widget_set_sensitive(self->menuEntryApply, info.info->page.format != PageTypeFormat::Copy);
```